config/link-check/config.yml

```diff
@@ -0,0 +1,12 @@
+rootURL: https://dvc.org
+fileIncludePatterns: '{.github,content,src}/**/*.{css,js,jsx,md,tsx,ts,json}'
+fileExcludePatternFiles: config/link-check/excluded-files.yml
+linkExcludePatternFiles: config/link-check/excluded-links.yml
+bottlenecks:
```

Comment:
let's call it custom link handler or something, bottleneck sounds a bit strange. Also, it might happen we'll want to define some additional params here per link type/domain

Reply:
This is now renamed to `linkOptions`